include/nighthawk/adaptive_load/nighthawk_service_client.h

```diff
@@ -0,0 +1,38 @@
+#pragma once
+#include "envoy/common/pure.h"
+
+#include "external/envoy/source/common/common/statusor.h"
+#include "external/envoy/source/common/protobuf/protobuf.h"
+
+#include "api/client/options.pb.h"
+#include "api/client/service.grpc.pb.h"
+
+namespace Nighthawk {
+
+/**
+ * An interface for interacting with a Nighthawk Service gRPC stub.
+ */
+class NighthawkServiceClient {
```

Comment:
Curious what you think of this train of thought:

At first I was going to ask if you could rename this to be Adaptive Load specific, since NighthawkServiceClient is the kind of name that could be used by a future class in Nighthawk, for creating a generic client for the NighthawkService.

Then I looked at the code, and I don't think there's anything adaptive-load specific about this class or the impl

Reply:
You're right, moved it to `common/`.